components/02-elements/form/select/Select.stories.js

```diff
@@ -0,0 +1,49 @@
+import {storiesOf} from '@storybook/vue'
+
+import countries from './mocks/countries.json'
+
+import App from '../../../01-globals/app/App.vue'
+import AlpacaSelect from './Select.vue'
+import AlpacaLabel from '../../../01-globals/label/Label.vue'
+
+storiesOf('Elements/Form/Select', module)
+  .add('Default', () => ({
+    components: { App, AlpacaSelect, AlpacaLabel },
+    template: `
+      <app>
+        <alpaca-select
+          :id="field.id"  
+          :name="field.name"
+          :options="countryOptions"
+        >
+          <alpaca-label
```

Comment:
I think, alpaca-label shoud be inside alpaca-select component (just like in input component)